crates/api/src/entrypoint.rs

```diff
@@ -1,30 +1,157 @@
 use crate::{deploy::DeployFile, manifest::WasmImage, wasmtime_unit::Wasmtime};
 
+use std::env;
 use std::path::{Component, Path, PathBuf};
 
 use anyhow::{bail, Result};
 use log::info;
 
-/// Validates the deployed image.
-///
-/// Takes path to the workdir as an argument.
-pub fn start(workdir: impl AsRef<Path>) -> Result<()> {
-    let workdir = workdir.as_ref();
-    let deploy_file = DeployFile::load(workdir)?;
+const INIT_MEM_VAR: &str = "YA_RUNTIME_WASI_INIT_MEM";
+const OPTIMIZE_VAR: &str = "YA_RUNTIME_WASI_OPT";
+const SGX_VAR: &str = "YA_RUNTIME_WASI_SGX";
 
-    info!(
-        "Validating deployed image {:?}.",
-        get_log_path(workdir, deploy_file.image_path())
-    );
+/// [Yagna] WASI runtime configuration.
+#[derive(Default, Clone, Debug)]
+pub struct RuntimeOptions {
+    pub(crate) max_static_memory: Option<u64>,
+    pub(crate) optimize: Option<bool>,
+    pub(crate) sgx_profile: Option<bool>,
+}
 
-    let mut image = WasmImage::new(&deploy_file.image_path())?;
-    let mut wasmtime = create_wasmtime(workdir, &mut image, &deploy_file)?;
+impl RuntimeOptions {
+    /** Initializes runtime options from environment variables.
+     *
+     * * `YA_RUNTIME_WASI_INIT_MEM` - maximum memory size. (supported formats 250m, 1.2g)
+     * * `YA_RUNTIME_WASI_OPT` - optimization. (0|no for no optimalization), (1|yes)
+     * * `YA_RUNTIME_WASI_SGX` - enables sgx profiled configuration.
+     */
```

Comment:
Can we settle on one style of documenting methods, i.e., either strive to use `///` everywhere, or `/** */`?